packages/aws-cdk/lib/cli.ts

```diff
@@ -255,7 +256,8 @@ async function parseCommandLineArguments(args: string[]) {
     .command('destroy [STACKS..]', 'Destroy the stack(s) named STACKS', (yargs: Argv) => yargs
```

Comment:
would it make sense to also add an option for `cdk watch`??

Reply:
Correct me if I am wrong but AFAIK `cdk watch` essentially monitors a stack and update it through SDK calls without redeploying it. I think `cdk watch` would not monitor the stack status so I can't see how this would fit in this scenario. I might be wrong though.

Reply:
Ah, I thinkk we're both right. It's an options to turn off the --hot-swap for `watch` https://docs.aws.amazon.com/cdk/v2/guide/cli.html#cli-deploy.

This comment is just a suggestion, though. I would approve this PR with our without watch support, since that could be added in the future, if there was a demand for it